src/Stryker.CLI/Stryker.CLI/CommandLineConfig/CommandLineConfigReader.cs

```diff
@@ -136,8 +136,8 @@ private void PrepareCliOptions(IStrykerInputs inputs)
             AddCliInput(inputs.DashboardApiKeyInput, "dashboard-api-key", null, category: InputCategory.Reporting);
             AddCliInput(inputs.AzureFileStorageSasInput, "azure-fileshare-sas", null, category: InputCategory.Reporting);
 
+            AddCliInput(inputs.TestsFailureBreakInput, "tests-failure-break", null, optionType: CommandOptionType.SingleOrNoValue, category: InputCategory.Misc);
```

Comment:
What do you think of this for the name

```suggestion
            AddCliInput(inputs.BreakOnInitialTestFailure, "break-on-intitial-test-failure", null, optionType: CommandOptionType.SingleOrNoValue, category: InputCategory.Misc);
```

Reply:
Fixed on new commits